apps/dashboard/app/assets/stylesheets/common.scss

```diff
@@ -1,3 +1,6 @@
+#loading-icon {
+  text-align: center;
+}
```

Comment:
I'm not in love with CSS classes identified through ids.  Plus I think you can just use the bootstrap class `text-center`.